controllers/imagejob/imagejob_controller.go

```diff
@@ -158,7 +158,7 @@ func checkNodeFitness(pod *corev1.Pod, node *corev1.Node) bool {
 //+kubebuilder:rbac:groups=eraser.sh,resources=imagejobs,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups="",resources=podtemplates,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=eraser.sh,resources=imagejobs/status,verbs=get;update;patch
-//+kubebuilder:rbac:groups="",resources=configmaps,verbs=get;list;watch
+//+kubebuilder:rbac:groups="",resources=configmaps,verbs=get;list;watch;create;update;patch;delete
```

Comment:
@pmengelbert looks like create and update are needed, do we need `delete`?

Reply:
Yeah I believe so.